### PR TITLE
Adjust restrictions

### PR DESCRIPTION
**Pull request by OCGDFS**
- matched cybran and uef restrictions
- (Objectives.KillOrCapture() doesnt work)
- replaced RAS with Gun

### CODE REVIEW

File: SCCA_Coop_E01/SCCA_Coop_E01_script.lua [hunk 1]
```diff
@@ -119,11 +119,11 @@ function OnStart(self)
             ScenarioFramework.SetArmyColor(ScenarioInfo[army], unpack(color))
         end
     end
-    
-    -- Disable friendly AI sharing resources to players
-    GetArmyBrain(EastResearch):SetResourceSharing(false)
+	    
+	-- Disable friendly AI sharing resources to players
+	GetArmyBrain(EastResearch):SetResourceSharing(false)
```
Comment (TheKeyBlue) [inline SCCA_Coop_E01/SCCA_Coop_E01_script.lua]: This added space needs to be removed.

File: SCCA_Coop_E01/SCCA_Coop_E01_script.lua [hunk 2]
```diff
@@ -530,33 +541,28 @@ end
 -- Mission 5
 -----------
 function IntroMission5()
-    ScenarioInfo.MissionNumber = 5
+	ScenarioInfo.MissionNumber = 5
 
-    ScenarioFramework.SetSharedUnitCap(480)
+	ScenarioFramework.SetSharedUnitCap(480)
```
Comment (TheKeyBlue) [inline SCCA_Coop_E01/SCCA_Coop_E01_script.lua]: Same here

**Comment by TheKeyBlue** [comment]
what do you mean with (Objectives.KillOrCapture() doesnt work) ?
Are you going to fix this in future commits in the same PR?

And its best not to fix Objectives.KillOrCapture in a PR that says: Adjust restrictions.
So if you're going to fix it in this PR then rename it to something more general.
like : UEF M1: fix restrictions and objectives bugs

**Comment by OCGDFS** [comment]
this was just to make clear that u can capture all cybran buildings, but not buildings which you have to kill.

**Comment by OCGDFS** [comment]
the spaces are somehow wrong added by my editor, can i change the request somehow, or do i have to make a new one?

**Comment by TheKeyBlue** [comment]
you can amend your latest commit
https://www.atlassian.com/git/tutorials/rewriting-history/git-commit--amend

I think we use 4 spaces for each tab, so you should set your editor to use that format.

**Comment by OCGDFS** [comment]
> >this was just to make clear that u can capture all cybran buildings, but not buildings which you have to >kill.

>A little confusing to me. So you can't capture buildings that you have to kill? But KillOrCapture allows you to choose one or the other? So how does that apply there?

the goal was to match cybran and uef restrictions. If you wanted to capture a cybran unit, it could happen, that the unit disappeared because u didnt have the permission to build that unit. I changed the permissions now, but it doesnt work for units you have to KillOrCapture, because you just are not able to capture them.

**Comment by TheKeyBlue** [comment]
okay i took a look at the script and you are right, you can't capture them, but thats not your fault.
The script made it impossible. (looking at the objectives, they talk about destroying, so not sure why KillOrCapture instead of Kill)
See:
https://github.com/FAForever/faf-coop-maps/blob/master/SCCA_Coop_E01/SCCA_Coop_E01_script.lua#L461
https://github.com/FAForever/faf-coop-maps/blob/master/SCCA_Coop_E01/SCCA_Coop_E01_script.lua#L561

**Comment by OCGDFS** [comment]
close it. pulled new request

**Pull request closed without merging**